src/app/views/components/table/table.component.html

```diff
@@ -7,7 +7,7 @@ <h3 class="forge-typography--subtitle1">
       </h3>
       <forge-table select="true"
                  [columnConfigurations]="columnConfigurations"
-                 [data]="data$ | async"
```

Comment:
`data` doesn't currently allow `null`, but the `async` pipe can always potentially return null.  I also did `?? []` in a different example below.  But also potentially a case where we should expand the Forge type if nullish values are actually being handled properly.